### PR TITLE
Tg feature userauthasync

### PR DESCRIPTION
**Pull request by talgrewal**
# Feature userauthasync

- Add User Token for sign up and login
- Add signout button for side drawer

### CODE REVIEW

File: client/js/components/AccountLoginForm/AccountLoginForm.js [hunk 1]
```diff
@@ -47,16 +55,14 @@ class AccountLoginForm extends Component {
...
                     },
                   });
-
-                  if (data) {
-                    this.props.navigation.navigate('Home');
-                  }
+                  await createViewer(newUserToken.data.login);
+                  this.props.navigation.navigate('Home');
                 } catch (e) {
                   console.log(e);
```
Comment (siddharthparmar7) [inline client/js/components/AccountLoginForm/AccountLoginForm.js:67]: Display errors if any for your user

File: client/js/components/AccountLoginForm/AccountLoginForm.js [hunk 2]
```diff
@@ -111,7 +117,6 @@ class AccountLoginForm extends Component {
 
                   {/* <MainSigninButton /> */}
```
Comment (siddharthparmar7) [inline client/js/components/AccountLoginForm/AccountLoginForm.js]: remove commented code.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work. Please make the changes. Thank you.

File: client/js/components/AccountLoginForm/AccountLoginForm.js (hunk 2 above)
Comment (talgrewal) [reply client/js/components/AccountLoginForm/AccountLoginForm.js]: done

File: client/js/components/AccountLoginForm/AccountLoginForm.js (hunk 1 above)
Comment (talgrewal) [reply client/js/components/AccountLoginForm/AccountLoginForm.js:67]: This is done. Error text added to sign up and login form based on errors.

**Comment by talgrewal** [comment]
This is done.

**Comment by talgrewal** [comment]
Ready for Review

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**